controllers/subnamespace_controller.go

```diff
@@ -41,14 +42,16 @@ type SubnamespaceReconciler struct {
 
 func (r *SubnamespaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := r.Log.WithValues("subspace", req.NamespacedName)
-	var subspace danav1alpha1.Subnamespace
-	var ownerNamespace v1.Namespace
-	var childNamespace v1.Namespace
-	var childNamespaceName string
+	var (
+		ownerNamespace v1.Namespace
+		subspace       danav1alpha1.Subnamespace
+	)
+
+	log.Info("starting to reconcile")
```

Comment:
Debug log might fit better for non-active logs (like "reconcile start / stop" and functions calls).

Reply:
I dont thing` loger.Logger` has a Debug method